m0t1fs/linux_kernel/st/m0t1fs_common_inc.sh

```diff
@@ -96,6 +96,8 @@ SNS_MOTR_CLI_EP="12345:33:1000"
 POOL_MACHINE_CLI_EP="12345:33:1001"
 SNS_QUIESCE_CLI_EP="12345:33:1002"
 M0HAM_CLI_EP="12345:33:1003"
+FDMI_PLUGIN_EP="12345:33:1004"
+FDMI_FILTER_FID="6c00000000000001:0"
```

Comment:
This Codacy warning is false-positive.

It is used in another file when this file is included there.